list-ops/list_ops_test.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+import math
+
+from list_ops import map_src, length_src, reduce_src
+
+
+class ListOpsTest(unittest.TestCase):
+
+    # necessary functions to test map_src
+    def square(self, x):
+        return x**2
```

Comment:
Short functions like this one can be more concisely (and elegantly?) expressed with lamdbas.

E.g. `map(lambda x: x**2, [-1, 2, ...])`.